desktop/src/main.rs

```diff
@@ -64,6 +64,13 @@ fn init() {
         panic_hook(info);
     }));
 
+    // Redirect Log to Tracing if it isn't already
+    let _ = tracing_log::LogTracer::builder()
+        // wgpu crates are extremely verbose
+        .ignore_crate("wgpu_hal")
```

Comment:
We do this because there's no equivalent to `RUST_LOG` on web. It should be up to the user if they want, or don't want, to see wgpu things.